src/status_im/components/drawer/view.cljs

```diff
@@ -34,60 +34,70 @@
 (defn open-drawer [] (.openDrawer @drawer-atom))
 (defn close-drawer [] (.closeDrawer @drawer-atom))
```

Comment:
What about calling this `open-drawer!` to signify side effect? Same with `close-drawer` and possibly `save-profile`.